src/components/user-table/user-table.tsx

```diff
@@ -33,15 +31,9 @@ export const UserTable = () => {
 			className="user-table"
 		>
 			<UserTableHeader orderByData={orderByData} onOrderByChanged={handleOnOrderByChanged} />
-			<Show if={isNotStartedOrPending(brukereFetcher)}>
-				<Spinner />
-			</Show>
+			{isNotStartedOrPending(brukereFetcher) && <Spinner />}
 			{tableBrukere.length === 0 && hasFinished(brukereFetcher) ? (
-				<AlertStripeInfo className="user-table__no-users">
-					<span role="alert" aria-live="polite">
-						Fant ingen brukere
-					</span>
-				</AlertStripeInfo>
+				<Alert variant="info">Fant ingen brukere</Alert>
```

Comment:
Dersom `tableBrukere.length === 0 && hasFinished(brukereFetcher)` evaluerer til `false` så vil `<UserTableBody ...>` fremdeles vises, selv om vi også viser spinner. Er dette hensikten?

Reply:
Hmm, det går vel greit? `<UserTableBody />` har ingen høyde når den ikke har innhold.

Reply:
Joda, men gir kanskje ikke så mye mening å rendre en komponent (sånn conditional-rendering-messig) når vi uansett ikke har innhold å gi den - jeg tror det kan være litt mer forutsigbart for leseren, men det er ikke _så_ farlig.